src/rust/ensogl/src/math/circle_segment.rs

```diff
@@ -0,0 +1,55 @@
+//! Provides functionality related to circle segments.
+use crate::math::topology::unit::{Angle, Radians};
+
+
+
+// =====================
+// === CircleSegment ===
+// =====================
+
+/// Implements computations related to circle segments.
+/// For details and background on the formulas used, see
+/// https://en.wikipedia.org/wiki/Circular_segment
+#[derive(Clone,Copy,Debug,PartialEq)]
+pub struct CircleSegment{
```

Comment:
A very minor thing, but we put spaces before `{` everywhere. Also before and after `:` in struct definitions (lines below).